go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/giantswarm/apiextensions/v3 v3.22.0
 	github.com/giantswarm/apptest v0.10.3
 	github.com/giantswarm/backoff v0.2.0
-	github.com/giantswarm/k8sclient/v5 v5.11.0
+	github.com/giantswarm/k8sclient/v5 v5.11.1-0.20210602043110-eb11ba8aa424
```

Comment:
Once I've tested this, I'll make a new release of `k8sclient` and update this to use it.